### PR TITLE
Fix unicode in bulk app translations

### PR DESCRIPTION
**Pull request by orangejenny**
Caused by https://github.com/dimagi/commcare-hq/pull/8523 yesterday

Added unicode tests to upload & download.

@nickpell

### CODE REVIEW

**Comment by orangejenny** [comment]
would like to get in before today's deploy

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ class BulkAppTranslationBasicTest(BulkAppTranslationTestBase):
           ("question3/question4-label", 'question6: <output value="/data/question6"/>', 'question6: <output value="/data/question6"/>', "", "", "", "", "", ""),
           ("question3/question5-label", "English Label", "English Label", "", "", "", "", "", ""),
           ("question7-label", 'question1: <output value="/data/question1"/> &lt; 5', "question7", "", "", "", "", "", ""),
-          ('add_markdown-label', 'add_markdown: ~~new markdown~~', 'add_markdown: ~~new markdown~~', '', '', '', '', '', ''),
+          ('add_markdown-label', 'add_markdown: ~~new \u0939\u093f markdown~~', 'add_markdown: ~~new \u0939\u093f markdown~~', '', '', '', '', '', ''),
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: E501 line too long (151 > 115 characters)

Comment (orangejenny) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: ignore

**Comment by orangejenny** [comment]
for future generations: ticket was http://manage.dimagi.com/default.asp?183957

**Comment by benrudolph** [comment]
Did this break tests?
```
======================================================================
FAIL: test_no_change (corehq.apps.app_manager.tests.test_bulk_ui_translation.BulkUiTranslation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_bulk_ui_translation.py", line 25, in test_no_change
    dict(translations), dict()
AssertionError: {u'en': {u'menu.basic': u'Basic Mode', u'odk_activity_not_found': u'No activity  [truncated]... != {}
Diff is 3995 characters long. Set self.maxDiff to None to see it.
----------------------------------------------------------------------
```

**Comment by benrudolph** [comment]
(no i don't think it did, just went back in history and that test didn't pass even before this was merged)

**Comment by orangejenny** [comment]
is that the test failure @dannyroberts fixed with https://github.com/dimagi/commcare-hq/pull/8640 ?

**Comment by benrudolph** [comment]
aha this explains why going back in history and not updating my submodules didn't fix the test. i think danny's fix was a short term solution, but i imagine if we were to update that submodule again the tests would break. anyways, i'm guessing the fix is in that submodule not in this PR

**Comment by benrudolph** [comment]
nvm, it looks totally fixed, sorry still catching up on email

**Pull request merged**